cluster/aws/coreos/util.sh

```diff
@@ -30,8 +30,8 @@ function detect-minion-image (){
 
 function generate-minion-user-data() {
   i=$1
+  # TODO(bakins): Is this actually used?
```

Comment:
@bakins I can't see anywhere MINION_PRIVATE_IP is used?  Is there somewhere I'm missing.

Also, heads up on this patch.  Shouldn't affect CoreOS, but you never know!  My plan is to finish the stream on AWS/Ubuntu and then fix what I've broken on AWS/CoreOS!

What I'm working towards here is putting all the minions in an AWS auto-scaling group (like GCE does)